packages/paste-core/components/combobox/src/Combobox.tsx

```diff
@@ -235,33 +235,30 @@ const Combobox = React.forwardRef<HTMLInputElement, ComboboxProps>(
           <ComboboxInputWrapper {...getComboboxProps({role: 'combobox'})}>
             <StyledInputAsSelect
               {...getToggleButtonProps({tabIndex: 0})}
-              {...getInputProps({disabled})}
+              {...getInputProps({disabled, ref})}
               {...(!autocomplete ? {onChange: event => event.preventDefault()} : undefined)}
               aria-describedby={helpTextId}
               {...props}
               type="text"
-              ref={ref}
               paddingRight="space90"
             />
             <SelectIconWrapper>
               <ChevronDownIcon aria-hidden="true" decorative color={iconColor} size="sizeIcon30" />
             </SelectIconWrapper>
           </ComboboxInputWrapper>
         </FormControlWrapper>
-        {isOpen && (
-          <ComboboxListbox {...getMenuProps()}>
-            {renderListBox({
-              items,
-              getItemProps,
-              highlightedIndex,
-              optionTemplate,
-              groupItemsBy,
-              groupLabelTemplate,
-              optionUID,
-              groupUID,
-            })}
-          </ComboboxListbox>
-        )}
+        <ComboboxListbox hidden={!isOpen} {...getMenuProps()}>
```

Comment:
Is the hidden prop the same as the hidden html attribute here?

Reply:
Yes, exactly the same. V6 expects the listbox to be rendered and warns if it's not, which to be fair is correct. This is just a nicer way to do it and hide the shadow we added to the listbox.